Stronger/Info/SummaryView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Summary.swift
+//  Stronger
+//
+//  Created by Theodore Kanell on 2/1/24.
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct SummaryView: View {
+    static var accountEnabled: Bool {
+        !FeatureFlags.disableFirebase && !FeatureFlags.skipOnboarding
+    }
+
+   
+    @State private var presentingAccount = false
+    
+    // var username: String
+    @State private var firstName: String = ""
+    @State private var lastName: String = ""
+    @State private var exerciseValue: Double = 50
+    @State private var dietValue: Double = 80
+    @State private var totalProtein: Double = 66
+
+    var body: some View {
+        VStack {
+            // Welcome message
+            Text("Welcome Mary")
```

Comment:
This looks like hard coded text, please be sure that you pull the username from the Spezi Account component.